packages/chakra-icons/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "dependencies": {
     "@ri7nz/gittar": "0.1.1-patch.3",
-    "create-chakra-icons": "*",
+    "create-chakra-icons": "1.0.3-next.1",
```

Comment:
I change this with following  https://classic.yarnpkg.com/lang/en/docs/workspaces/
![image](https://user-images.githubusercontent.com/16365952/148830972-7880cf3b-9c75-4997-a09f-25c0480950b0.png)

Reply:
LGTM. I don’t know why asterisks won’t work, I’ll investigate sometime later.